docs/src/pages/[platform]/components/autocomplete/examples/AutocompleteThemeExample.tsx

```diff
@@ -6,6 +6,7 @@ const theme: Theme = {
   tokens: {
     components: {
       autocomplete: {
+        // @ts-ignore
         menu: {
```

Comment:
well this is a weird one! seems to be occurring because most of the token keys are required by AutocompleteTokens type:
https://github.com/aws-amplify/amplify-ui/blob/main/packages/ui/src/theme/tokens/components/autocomplete.ts#L18

Reply:
We should fix that.... that is a bug

Reply:
https://github.com/aws-amplify/amplify-ui/pull/3470